modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/release-notes/fragments/release-notes-sidetab/index.js

```diff
@@ -0,0 +1,239 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+/* eslint-disable @liferay/portal/no-global-fetch */
+/* eslint-disable no-undef */
+
+const fetchRequest = async (input) => {
+	const response = await fetch(input, {
+		headers: {
+			'Accept-Language': Liferay.ThemeDisplay.getBCP47LanguageId(),
+			'Cache-Control': 'max-age=30, stale-while-revalidate=30',
+			'x-csrf-token': Liferay.authToken,
+		},
+	});
+
+	return response.json();
+};
+
+const restArticle = async (requestId) => {
+	const journalArticleResponse = await fetchRequest(
+		`/o/headless-delivery/v1.0/structured-contents/${requestId}?fields=contentFields,relatedContents,taxonomyCategoryBriefs,title&nestedFields=embeddedTaxonomyCategory`
+	);
+
+	updateArticleContent(journalArticleResponse.contentFields);
+	updateArticleLinks(journalArticleResponse.contentFields);
+	updateArticleTitle(journalArticleResponse.title);
+	updateLabelProductCapabilities(
+		journalArticleResponse.taxonomyCategoryBriefs
+	);
+	updateLabelStatus(journalArticleResponse.taxonomyCategoryBriefs);
+};
+
+function createContentDiv(htmlContent) {
+	const divElement = document.createElement('div');
+
+	divElement.innerHTML = htmlContent;
+
+	return divElement;
+}
+
+function updateArticleContent(contentFields) {
+	const articleContent = document.getElementById('articleContent');
+
+	if (!articleContent) {
+		return;
+	}
+
+	articleContent.innerHTML = '';
+
+	contentFields.forEach((field) => {
+		if (field.contentFieldValue && field.contentFieldValue.data) {
+			const contentDiv = createContentDiv(field.contentFieldValue.data);
+
+			articleContent.appendChild(contentDiv);
+		}
+	});
+}
+
+function updateArticleLinks(contentFields) {
+	const articleLink = document.getElementById('articleLink');
+
+	if (articleLink) {
+		articleLink.innerHTML = '';
+
+		contentFields.forEach((field) => {
+			let url = '';
+			let urlTitle = '';
+
+			field.nestedContentFields.forEach((nestedField) => {
+				const nestedData = nestedField.contentFieldValue?.data;
+
+				if (nestedData) {
+					if (nestedField.label.includes('Title')) {
```

Comment:
instead of nesting ifs we can write this with just one if else statement (the if can have two conditions